frontend/app/src/locales/cn.json

```diff
@@ -2055,8 +2055,8 @@
       "sync_enabled": "同步启用"
     },
     "inputs": {
-      "api_key": "API Key",
-      "api_secret": "API Secret",
+      "private_key": "Private Key",
```

Comment:
Why are you changing the kraken account related text? You should not touch these stuff for any locale.

Reply:
Apologies, eslint was throwing error. I am reverting this change and adding the changes specific to Coinbase's new Keys

Reply:
it's not fixed. Taking over.